### ISSUE TITLE
Windows branch in run_lcov_on_directory should return proc.stdout

### ISSUE THREAD

**Issue by marco-c** [klahnakoski/TestLog-ETL#26]
Like the Linux branch, it should return proc.stdout instead of calling parse_lcov_coverage.
The caller of run_lcov_on_directory will iterate proc.stdout.

**Comment by klahnakoski** [comment]
grcov requires lcov, and lcov does not work on windows. Keep this issue open until we have a way to test windows and write code for it.

**Comment by marco-c** [comment]
`grcov` doesn't require lcov, it only requires gcc.

**Comment by klahnakoski** [comment]
oh yes, I misspoke

    let status = Command::new("gcov")

**Comment by marco-c** [comment]
Yep, that should be part of gcc. If you have gcc installed, gcov should be available.

**Comment by klahnakoski** [comment]
"If you have gcc installed" is a difficult condition to meet on window.

**Comment by marco-c** [comment]
Have you tried with MinGW?

**Comment by klahnakoski** [comment]
I tried msys64, and I assume MinGW to be the same: The complexity is at the point where a windows python process tries to call a program in a linux vm.
